src/main/java/core/basesyntax/dataprocessor/DataProcessorStrategyImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.dataprocessor;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DataProcessorStrategyImpl implements DataProcessorStrategy {
```

Comment:
The same. Remove this class. Don't base your solution on your own guesses about future extensions, it leads to a lot of unnecessary code that will never be used.
Follow the KISS principle